gnd/src/main/java/com/google/android/gnd/ui/editobservation/EditObservationViewModel.java

```diff
@@ -193,6 +204,44 @@ public void onFocusChange(Field field, boolean hasFocus) {
     }
   }
 
+  void initPhotoSelector(String fieldId) {
```

Comment:
"init" may imply it sets up the photo selector, but doesn't actually show it. One clearer name could be "showPhotoSelector()"